ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -280,8 +280,8 @@ export default class TaskRunsCard extends Component {
                 {!artifact.isPublic && artifact.url && <OpenInNewIcon />}
               </TableCell>
             </Link>
-          </TableRow>
-        )}
+          </TableRow>;
+        }}
```

Comment:
This does indeed remove the error but also removes all of the artifacts from the UI since the renderRow function doesn't return anything anymore.

#### Before

<img width="800" alt="Screen Shot 2019-10-09 at 4 54 01 PM" src="https://user-images.githubusercontent.com/3766511/66519519-77b76a80-eab5-11e9-8b2e-64283d67bc37.png">


#### After

<img width="783" alt="Screen Shot 2019-10-09 at 4 53 53 PM" src="https://user-images.githubusercontent.com/3766511/66519518-77b76a80-eab5-11e9-97ca-d932b7f36c5f.png">